src/data/faq.json

```diff
@@ -116,6 +116,62 @@
+                "accordion": [
+                    {
+                        "title": "What is the purpose of the digital COVID test certificate?",
+                        "anchor": "test_cert_why",
+                        "active": true,
+                        "textblock": [
+                            "Starting with version 2.4 of the Corona-Warn-App (CWA), you can request a digital test certificate via the app.  The new feature allows you to prove your negative test result through an official, digital COVID test certificate in the form of a QR code in the CWA.  Within the countries of the European Union, it can serve as officially valid proof of a negative test result, which you can use, for example, for travel or in other legally prescribed cases, such as a visit to a restaurant. You can then show the QR code in the CWA."
+                        ]
+                    },
+                    {
+                        "title": "How can I request a test certificate?",
+                        "anchor": "test_cert_get",
+                        "active": true,
+                        "textblock": [
+                            "You can request the certificate by registering a test in the Corona-Warn-App (CWA): To do this, you can tap on 'Register Test' on the star screen of your CWA and then on 'Scan QR Code'. After scanning the QR code for test registration, the window for the COVID test certificate automatically opens. There you  can 'request the test certificate' or reject it if you do not want to receive one."
+                        ]
+                    },
+                    {
+                        "title": "Can I request test certificates for rapid tests and PCR tests?",
+                        "anchor": "test_cert_rat_and_pcr",
+                        "active": true,
+                        "textblock": [
+                            "You can request a certificate for both PCR and rapid tests, which is then issued in the event of a negative test.",
+                            "<b>Rapid tests</b><br>In the case of rapid tests, it may be that the respective test site does not support the digital certificate. In this case, after scanning the QR code, you will not be redirected to apply for the certificate. You will then receive the information that a test certificate cannot be requested because the test site does not support the issuance of test certificates.","<b>PCR tests</b><br>When requesting a test certificate for a PCR test, you must provide your date of birth. It is important that the date of birth corresponds to the one that was also indicated for the sample of PCR test. If the dates of birth do not match, for example due to a typo, you will not receive the test result or the certificate via the app. You cannot change the date of birth that you enter in the app afterwards. Before you request the test certificate after entering the date, you should therefore briefly check whether the date is correct."

```

Comment:
"of PCR test" => "of the PCR test"

Reply:
The text says that if the dob don't match "[…] you will not receive the **test result** or the certificate via the app". Is it really true that also the test result will not be shown in the app if the dob don't match? I thought that you just can't request a certificate in this case...

Reply:
ok